lucene/build.gradle

```diff
@@ -29,7 +29,7 @@ spotless {
 
 // spotless:off
 def repoToSha = [
-	'us-presidential-debates', 'a3ce5d43ce931858345b22d25d35344e0228e123'
+	'us-presidential-debates', '877ede3aea28a202c94bd169a8d1376624c3397a'
```

Comment:
**1)** git hash of the `v1.0.0` commit of https://github.com/mytakedotorg/us-presidential-debates